recipes/coordinatesconverter/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "coordinatesconverter" %}
+{% set version = "0.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/CoordinatesConverter-{{ version }}.tar.gz
+  sha256: ea0348c97569a6aaffac6406bdb4a260ea5f3e141d52a88cbdc9bb638667e494
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - CoordinatesConverter
+  commands:
+    - pip check
+  requires:
+    - pip
+    - numpy
+    - shapely
```

Comment:
Why are they only listed here and not in `run`?

Reply:
Sorry, my mistake. They are now correct.